queries.sql

```diff
@@ -99,3 +99,53 @@ SELECT owners.full_name,
 	ORDER BY total_animals
 	DESC LIMIT 1;
 SELECT * FROM animals;
+
+SELECT animals.name,
+    owners.full_name
+    FROM animals
+    JOIN owners
+    ON animals.owners_id = owners.id
+    WHERE full_name = 'MelodyPond';
+
+SELECT animals.name AS animal_name,
+    species.name
+    AS species_type
+    FROM animals
+    JOIN species
+    ON animals.species_id = species.id
+    WHERE species.name='Pokemon';
+
+SELECT full_name,
+    animals.name
+    FROM owners
+    LEFT JOIN animals
+    ON animals.owners_id=owners.id;
+
+SELECT count(animals)
+    FROM animals
+    JOIN species
+    ON animals.species_id=species.id;
+
+SELECT animals.name
+    FROM animals
+    JOIN owners
+    ON animals.owners_id=owners.id
+    WHERE owners.full_name='Jennifer Orwell';
+
+SELECT animals.name, 
+    animals.escape_attempts
+    FROM animals
+    INNER JOIN owners
+    ON owners.id = animals.owners_id
+    WHERE owners.full_name = 'Dean Winchester'
+    AND animals.escape_attempts = 0;
+
+SELECT owners.full_name, 
+    COUNT(animals.name)
+    AS total_animals
+    FROM owners
+    JOIN animals
+    ON animals.owners_id=owners.id
+    GROUP BY owners.full_name
+    ORDER BY total_animals
+    DESC LIMIT 1;
```

Comment:
- [ ] Kindly check the list of requirements in the screenshot below and ensure that they are implemented in `queries.sql` as per the project requirements.
![Screenshot from 2022-01-13 22-42-50](https://user-images.githubusercontent.com/20245164/149399375-9eb65d83-8a71-43c4-a9be-d2f13de0c555.png)